### PR TITLE
Add TravisCI integration

### PR DESCRIPTION
**Pull request by sprsquish**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -21,3 +21,4 @@ RDoc::Task.new do |rdoc|
 end
 
 task prerelease: [:test, :install]
+task :default => [:test]
```
Comment (ashfire908) [inline Rakefile]: `task default: [:test]`, to match the line above.

Comment (ashfire908) [inline Rakefile]: Actually you might be able to omit the `[]` as well.

**Comment by sprsquish** [comment]
how's this?

**Comment by ashfire908** [comment]
How come one of the builds failed?

**Comment by ashfire908** [comment]
LGTM

**Pull request merged**